packages/apps/plugins/plugin-graph/src/graph/graph.ts

```diff
@@ -47,149 +52,4 @@ export class GraphStore implements Graph {
       this.traverse({ from: from.parent, direction, predicate, onVisitNode });
     }
   }
-
-  registerNodeBuilder(builder: Graph.NodeBuilder): void {
-    this._nodeBuilders.add(builder);
-  }
-
-  removeNodeBuilder(builder: Graph.NodeBuilder): void {
-    this._nodeBuilders.delete(builder);
-  }
-
-  // TODO(burdon): Separate builder from Graph interface.
-  construct(from = this._root as Graph.Node, path: string[] = [], ignoreBuilders: Graph.NodeBuilder[] = []): void {
-    this._index[from.id] = path;
```

Comment:
The reason the test is failing is because this is missing from the new impl. The index is never being updated so graph nodes can't be looked up.